superbench/benchmarks/micro_benchmarks/cuda_nccl_bw_performance.py

```diff
@@ -188,9 +191,13 @@ def _process_raw_result(self, cmd_idx, raw_output):    # noqa: C901
                         busbw_out = float(line[busbw_index])
                         time_out = float(line[time_index])
                         algbw_out = float(line[algbw_index])
-                        self._result.add_result(self._args.operation + '_' + str(size) + '_busbw', busbw_out)
-                        self._result.add_result(self._args.operation + '_' + str(size) + '_algbw', algbw_out)
-                        self._result.add_result(self._args.operation + '_' + str(size) + '_time', time_out)
+                        exec_index = '_{}_{}_'.format(
+                            int(serial_index), int(parallel_index)
+                        ) if serial_index and parallel_index else '_'
+                        prefix_name = '{}{}{}_'.format(self._args.operation, exec_index, size)
```

Comment:
can you use ":" instead of "_" to separate index and size? it will be easier to match in diagnosis

Reply:
Just for double-check, do you mean the naming rule like:  `nccl-bw/${operation}_${serial_index)_${parallel_index):${msg_size}_time` e.x.: allreduce_0_0:1073741824_time?

Reply:
yep